test/unit/dom/dom.js

```diff
@@ -983,4 +983,56 @@ describe('skatejs-named-slots dom', function () {
       expect(elem.parentNode).to.equal(frag);
     });
   });
+
+  describe('Return elements', function () {
```

Comment:
Do you think it might be better to test this in each methods respective tests instead?